src/release.sh

```diff
@@ -11,58 +11,63 @@ fi
 cd "$(dirname "$0")/.."
 mkdir -p "$ver"
 
-version=$(curl -sS https://api.github.com/repos/openshift/origin/releases \
+api_user=""
+if [ -n "${GITHUB_API_USER}" ]; then
+    api_user="${GITHUB_API_USER}@"
+fi
+
+version=$(curl -sS https://${api_user}api.github.com/repos/openshift/origin/releases \
   | jq --arg ver "$ver" --raw-output \
       '.[]| select((.prerelease|not) and (.tag_name|startswith($ver))) | .tag_name' \
   | head -n 1)
 
-helm_version=$(curl -sS https://api.github.com/repos/helm/helm/releases \
-  | jq --raw-output \
-      '.[]| select(.prerelease|not) | .tag_name' \
-  | sed '/v3\./d' \
-  | head -n 1)
+helm_version_2=v2.16.4
+helm_version_3=v3.1.2
```

Comment:
```suggestion
helm2_version==$(curl -sS https://api.github.com/repos/helm/helm/releases \
  | jq --raw-output \
      '.[]| select(.prerelease|not) | .tag_name' \
  | sed '/v3\./d' \
  | head -n 1)
helm2_version==$(curl -sS https://api.github.com/repos/helm/helm/releases \
  | jq --raw-output \
      '.[]| select(.prerelease|not) | .tag_name' \
  | sed '/v2\./d' \
  | head -n 1)
```
These two commands yield the same values to denoted above. Tried out myself just now in the terminal.